### ISSUE TITLE
Please add mail header with Loomio group to outgoing mail

### ISSUE THREAD

**Issue by madduck** [loomio/loomio#7671]
Could mail sent by Loomio (e.g. notifications) please include a `X-Loomio-Group` header or somesuch, allowing easy filtering?

**Comment by robguthrie** [comment]
yes, it will be the same value as what we prefix emails with eg: [group-handle]
